### DISCUSSION TITLE
I have hard time running getMeasuringContext(text, optionsopt) function

### DISCUSSION THREAD

**Discussion by klavsbuss** [fabricjs/fabric.js#7355] [Q&A] [answered]
Running `canvas.getContext('text')` does not work. What parameters exactly should I pass to it?
Also in documentation it says that i can pass `Options object` e.g.` getMeasuringContext(text, optionsopt)`, but it's not clear what exact parameters i can pass as an object? Help much appreciated 🙏

**Comment by i-prikot** [comment, marked as answer]
> canvas.getContext('text')

Why did you decide this should work?

http://fabricjs.com/docs/fabric.Canvas.html#getContext

**Reply by i-prikot** [reply]
You can call the method like this:

```
const text = new fabric.IText("your text");
text.getMeasuringContext();
```

Based on the source code, the method no longer accepts arguments.

**Reply by klavsbuss** [reply]
Ivan, thanks a lot!

**Reply by asturur** [reply]
getMeasuringContext is an internal function and is not supposed to do anything that is really useful.
I would suggest you don't use it.
I will update the docs
